Azure/sonic-buildimage
# sFlow : Upgrade hsflowd to v2.0.26-4

**- What I did**
As per SONiC CLI requirements, we wanted the collector port to default to 6343 if not explicitly configured. In the Click based CLI, if the collector port is omitted, the port will be set to 6343 by default (https://github.com/Azure/sonic-utilities/blob/master/config/main.py#L2543).
However, if sonic-mgmt-framework based CLI is used, hsflowd was using port 0 as default. host-sflow v2.0.26-4 addresses this issue (i.e. use 6343 in case “collector_port” is not set).

Upgrade hsflowd to v2.0.26-4 to pick up https://github.com/sflow/host-sflow/releases/tag/v2.0.26-4.
**- How I did it**
Update rules and makefile for sflow's hsflowd
**- How to verify it**
Using sonic-mgmt infra, configure a collector (without setting port) and check that the default port used by hsflowd is 6343.
**- Description for the changelog**

sFlow : Upgrade hsflowd to v2.0.26-4

**- A picture of a cute animal (not mandatory but encouraged)**
